Code/Tools/ProjectManager/Platform/Mac/ProjectExportWorker_mac.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectExportWorker.h>
+#include <ProjectManagerDefs.h>
+#include <ProjectUtils.h>
+
+#include <QDir>
+#include <QString>
+
+namespace O3DE::ProjectManager
+{
+    QString ProjectExportWorker::GetO3DECLIString() const
+    {
+        return QString("scripts/o3de.sh");
+    }
+    AZ::Outcome<QStringList, QString> ProjectExportWorker::ConstructKillProcessCommandArguments(const QString& pidToKill) const
```

Comment:
Interesting, using Qt to run a process to kill a process. C has built-in stuff do this already? I'm not sure. 
https://www.man7.org/linux/man-pages/man2/kill.2.html

Reply:
Even if it did, behavior might differ across platforms. That, and there's likely qt specific stuff that also has to get torn down as well. For that reason I prefer to keep with the existing builder code and use Qt to tear down the process.